### DISCUSSION TITLE
Nyugtatár adattárolásának időtartama

### DISCUSSION THREAD

**Discussion by laurelkft** [nav-gov-hu/eRECEIPT#13] [Kérdések és válaszok]
Nagyon fontos kérdésként merül fel a nyugtatás addatárolásának időtartama. 
Ugyan elhangzott, hogy garancialeveleket, és hasolnló dokumentumokat tárolhatunk a nyugtatárban, de manapság szép szép számmal vannak olyan termékek, melyek garanciája meghaladja az adózás miatti nyugta-, számlamegőrzés időtartamát.
Fontos szabályozni, hogy az eNyugták/pénztárgépes számlák mekkora időszakon át érhetőek el a Nyugtatárban.

**Comment by renced42** [comment]
A nyugtatár 6 évig tárolja a nyugtákat és a "garancia" adatokat. Ennyi időn belül lehet visszakérni. Amennyiben ezen időt meghaladóan van szükség az információra akkor annak rendelkezésre állásáról a vevőnek kell gondoskodnia.

**Reply by kobor42** [reply]
Mit jelent az, hogy garancia adat? Az egész github-os oldalon sehol nem merül fel a garancia fogalom, pedig szervesen illeszkedhetne az eNyugtához az eGaranciajegy.

**Comment by BatariZ** [comment]
A 6 év időközben 10 évre módosult.
Konkrét garanciajegy, mint kifejezés nem szerepel a fejlesztői dokumentációban. A beküldött Bizonylathoz tartozhat Vevői melléklet, mely garanciajegyként is használható.
